sdk/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonReader.java

```diff
@@ -126,6 +127,81 @@ public T readNextRow() throws IOException, InterruptedException {
     return formatDateAndTimeStamp((Object []) row);
   }
 
+  /**
+   * This method converts the date and timestamp columns into right format. Before conversion date
+   * is present as integer and timestamp is present as long. This method also flattens complex
+   * columns and format the date/timestamp child present in them.
+   *
+   * @param dimension
```

Comment:
Please define the params or else remove it

Reply:
Done